lagunita/lms/templates/instructor/instructor_dashboard_bottom.html

```diff
@@ -0,0 +1,8 @@
+<script>
+$(document).ready(function() {
+  $('.instructor-nav button').click(function() {
+    var path = (location.pathname + location.search + location.hash);
+    ga('send', 'pageview', path);
```

Comment:
@caesar2164 @tanyaxp 
Curiously, what does it mean that we're using a different invocation of `ga` here,
versus above in `google_analytics.html`?

The former passes a third argument `path` as a string,
whereas the latter passes a third argument as a _dictionary_ that contains `page: path`.

Reply:
good catch, @stvstnfrd, looks like the cleanest form is the string way: https://developers.google.com/analytics/devguides/collection/analyticsjs/pages#examples